jwql/tests/test_loading_times.py

```diff
@@ -48,6 +49,8 @@
     urls.append('{}/{}/'.format(instrument, rootname))
 
 
+@pytest.mark.skipif(os.path.expanduser('~') == '/home/jenkins',
+                    reason='Requires access to central storage.')
```

Comment:
Just looking at what part of this test requires access to the `config.json`, it's just getting the admin account username. Is that too much of a security risk that we couldn't just hard-code it into `get_base_url()`?

Reply:
@laurenmarietta I'm not 100% sure if that would or wouldn't break any sort of security policy, so I'm inclined to err on the side of caution 🤷‍♂️

Reply:
Okay, makes sense!